shared/components/ResourcesList/ResourcesList.js

```diff
@@ -172,7 +177,9 @@ function Resources({
       <Link
         className="link"
         onClick={_ =>
-          fixedPath
```

Comment:
I think we could use a function with early returns here, double ternary statement makes it _complicated_ instead of _complex_.